ocpmodels/trainers/dist_energy_trainer.py

```diff
@@ -170,11 +176,11 @@ def train(self):
                     loss = self._compute_loss(out, batch)
                 loss = self.scaler.scale(loss) if self.scaler else loss
                 self._backward(loss)
-                scale = self.scaler.get_scale() if self.scaler else 1.
+                scale = self.scaler.get_scale() if self.scaler else 1.0
 
                 # Compute metrics.
                 self.metrics = self._compute_metrics(
-                    out, batch, self.evaluator
+                    out, batch, self.evaluator, metrics={},
```

Comment:
This differs from how we do the metrics in the other trainers, `metrics=self.metrics`. Is this on purpose?

Reply:
Yeah, I did try making it consistent everywhere to reset to `{}` at regular intervals